fmriprep/workflows/bold/confounds.py

```diff
@@ -220,21 +204,29 @@ def init_bold_confs_wf(
     fdisp = pe.Node(nac.FramewiseDisplacement(parameter_source="SPM"),
                     name="fdisp", mem_gb=mem_gb)
 
-    # a/t-CompCor
-    mrg_lbl_cc = pe.Node(niu.Merge(3), name='merge_rois_cc', run_without_submitting=True)
-
-    tcompcor = pe.Node(
-        TCompCor(components_file='tcompcor.tsv', header_prefix='t_comp_cor_', pre_filter='cosine',
-                 save_pre_filter=True, save_metadata=True, percentile_threshold=.05,
-                 failure_mode='NaN'),
-        name="tcompcor", mem_gb=mem_gb)
-
+    # Generate aCompCor probseg maps
+    acc_masks = pe.Node(aCompCorMasks(is_aseg=freesurfer), name="acc_masks")
+
+    # Resample probseg maps in BOLD space via T1w-to-BOLD transform
+    acc_msk_tfm = pe.MapNode(ApplyTransforms(
+        interpolation='Gaussian', float=False), iterfield=["input_image"],
+        name='acc_msk_tfm', mem_gb=0.1)
+    acc_msk_brain = pe.MapNode(ApplyMask(), name="acc_msk_brain",
+                               iterfield=["in_file"])
+    acc_msk_bin = pe.MapNode(Binarize(thresh_low=0.99), name='acc_msk_bin',
+                             iterfield=["in_file"])
     acompcor = pe.Node(
         ACompCor(components_file='acompcor.tsv', header_prefix='a_comp_cor_', pre_filter='cosine',
                  save_pre_filter=True, save_metadata=True, mask_names=['combined', 'CSF', 'WM'],
```

Comment:
`aCompCorMasks` returns CSF, WM, Combined. Do we need to reorder like:

```suggestion
                 save_pre_filter=True, save_metadata=True, mask_names=['CSF', 'WM', 'combined'],
```

Reply:
Good catch 👍